doc/.vale.ini

```diff
@@ -23,7 +23,7 @@ Packages = Google
 Vocab = ANSYS
 
 # Apply the following styles
-[*.{md,rst}]
+[*.{rst}]
```

Comment:
This way CHANGELOG.md is excluded and no other markdown files are in the repository.